docs/merge_docs.sh

```diff
@@ -33,8 +33,8 @@ sed -i "s/md###t/html#t/g" demo.md
 sed -i 's/# Preparing/# /g' prepare_data.md
 sed -i 's/#/##&/' prepare_data.md
 
-sed -i '1i\## Action Localization Models' localization_models.md
-sed -i '1i\## Action Recognition Models' recognition_models.md
+sed -i '1i\# Action Localization Models' localization_models.md
```

Comment:
why change this line

Reply:
I put the contents of recognition_models and localization_models to model zoo.md, so that it can jump inside the page